docs/get-started-kubernetes.md

```diff
@@ -0,0 +1,136 @@
+# Getting Started on Kubernetes
```

Comment:
Nit: Since this is a different document now, just reading the heading tells that it's a general Kubernetes guide. Some suggestions for heading -

- Configure Kubernetes Using Coretools
- Getting started on configuring Kubernetes with coretools